openeo/rest/connection.py

```diff
@@ -1065,6 +1065,37 @@ def describe_process(self, id: str, namespace: Optional[str] = None) -> dict:
 
         raise OpenEoClientException("Process does not exist.")
 
+    """
+        Loads all available processes of the back end.
+
+        :param namespace: The namespace for which to list processes.
+
+        :return: Dictionary of All available processes of the back end.
+    """
+
+    def dict_processes(self, namespace: Optional[str] = None) -> dict:
+        processes = self.list_processes(namespace)
+        result = {}
+        for process in processes:
+            result[process["id"]] = process
+        return result
+
+    """
+        Returns schema of the parameter of the process from the back end.
+
+        :param process_id: The id of the process.
+        :param parameter_id: The id of the parameter.
+        :param namespace: The namespace of the process.
+
+        :return: schema of the parameter in the process.
+    """
+
+    def get_schema(self, process_id: str, parameter_id: str, namespace: Optional[str] = None) -> Union[dict, list]:
```

Comment:
This function name is too generic for a Connection method: there are more kinds of schema's in openEO than process parameter schema's